dojo/tools/deepfence_threatmapper/parser.py

```diff
@@ -0,0 +1,182 @@
+                    headers[row[i]] = i
+            elif headers.get("Rule Name") is not None and headers.get("Class") is not None:
+                Rule_Name = row[headers["Rule Name"]]
+                Class = row[headers["Class"]]
+                File_Name = row[headers["File Name"]]
+                Summary = row[headers["Summary"]]
+                Severity = row[headers["Severity"]]
+                Node_Name = row[headers["Node Name"]]
+                NodeType = row[headers["NodeType"]]
+                Container_Name = row[headers["Container Name"]]
+                Kubernetes_Cluster_Name = row[headers["Kubernetes Cluster Name"]]
+                description += "**Summary:** " + str(Summary) + "\n"
+                description += "**Rule Name:** " + str(Rule_Name) + "\n"
+                description += "**Class:** " + str(Class) + "\n"
+                description += "**File Name:** " + str(File_Name) + "\n"
+                description += "**Node Name:** " + str(Node_Name) + "\n"
+                description += "**NodeType:** " + str(NodeType) + "\n"
+                description += "**Container Name:** " + str(Container_Name) + "\n"
+                description += "**Kubernetes Cluster Name:** " + str(Kubernetes_Cluster_Name) + "\n"
+                findings.append(
+                    Finding(
+                        title=Rule_Name,
+                        description=description,
+                        file_path=File_Name,
+                        severity=self.severity(Severity),
+                        static_finding=False,
+                        dynamic_finding=True,
+                        test=test,
+                    )
+                )
+            elif headers.get("Filename") is not None and headers.get("Content") is not None:
+                Filename = row[headers["Filename"]]
+                Content = row[headers["Content"]]
+                Name = row[headers["Name"]]
+                Rule = row[headers["Rule"]]
+                Severity = row[headers["Severity"]]
+                Node_Name = row[headers["Node Name"]]
+                Container_Name = row[headers["Container Name"]]
+                Kubernetes_Cluster_Name = row[headers["Kubernetes Cluster Name"]]
+                Signature = row[headers["Signature"]]
+                description += "**Filename:** " + str(Filename) + "\n"
+                description += "**Name:** " + str(Name) + "\n"
+                description += "**Rule:** " + str(Rule) + "\n"
+                description += "**Node Name:** " + str(Node_Name) + "\n"
+                description += "**Container Name:** " + str(Container_Name) + "\n"
+                description += "**Kubernetes Cluster Name:** " + str(Kubernetes_Cluster_Name) + "\n"
+                description += "**Content:** " + str(Content) + "\n"
+                description += "**Signature:** " + str(Signature) + "\n"
+                if Name is not None and Severity is not None:
```

Comment:
@cneill: I have no clue here why I need this check, but otherwise, two additional findings will be added with all values "None".